### ISSUE TITLE
How to run the code

### ISSUE THREAD

**Issue by parthasarathy-eng** [takyonxxx/SdrTuner-RTL-SDR-RTL2832U#1]
I received the FM signal using GNU radio.Now i need top generate C code .
I need to know how I run the Cpp code provided by you.
Is it possible to take the real time input by using this code.
![12345](https://user-images.githubusercontent.com/83490383/165092332-ea708340-558f-4bb0-ab6a-7251a4aaa4a0.png)

**Comment by takyonxxx** [comment]
yes if you can compile it in linux and install neccassary libriraries it will work.
check .pro file for necceassary libs

________________________________
From: parthasarathy-eng ***@***.***>
Sent: Monday, April 25, 2022 3:50 PM
To: takyonxxx/SdrTuner-RTL-SDR-RTL2832U ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [takyonxxx/SdrTuner-RTL-SDR-RTL2832U] How to run the code (Issue #1)


I received the FM signal using GNU radio.Now i need top generate C code .
I need to know how I run the Cpp code provided by you.
Is it possible to take the real time input by using this code.
[12345]<https://user-images.githubusercontent.com/83490383/165092332-ea708340-558f-4bb0-ab6a-7251a4aaa4a0.png>

**Comment by takyonxxx** [comment]
i complied it for u
but you should run it in linux ubuntu in command terminal
type ./SdrRtlTuner

before u should do this steps

       sudo apt install libusb-1.0-0-dev

   sudo apt-get install -y fftw3-dev

   sudo apt install librtlsdr-dev

   sudo apt install libportaudio2

   sudo apt install portaudio19-dev

   sudo apt install sox

   sudo apt-get install libgl-dev

   sudo apt-get install qtmultimedia5-dev


________________________________

**Comment by parthasarathy-eng** [comment]
Ok sir
I will try

**Comment by freethinker-coder** [comment]
![1](https://user-images.githubusercontent.com/104486175/167348495-ac55b47b-8a00-4232-8ee8-9aa9553b9b1f.png)
 i tried this . I followed all the steps provided by you. But i obtain this .
How can i account this.

**Comment by takyonxxx** [comment]
type chmod +x SdrRtlTuner to make it executble
________________________________

**Comment by freethinker-coder** [comment]
Got this.
![new2](https://user-images.githubusercontent.com/104486175/167561987-822313eb-7a5d-4f6c-a1f0-b8a8f4ff3344.png)

**Comment by takyonxxx** [comment]
be sure that the file is in folder
________________________________
